driver/ppm_flag_helpers.h

```diff
@@ -1724,6 +1724,21 @@ static __always_inline u32 mlock2_flags_to_scap(unsigned long flags)
 	return res;
 }
 
+static __always_inline u32 memfd_create_flags_to_scap(unsigned long flags)
```

Comment:
The kernel provides us a u32 for flags so i would use it
```c
SYSCALL_DEFINE2(memfd_create,
		const char __user *, uname,
		unsigned int, flags)
```

```suggestion
static __always_inline u32 memfd_create_flags_to_scap(u32 flags)
```